chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -612,13 +612,12 @@ impl PeerManagerActor {
         self.add_verified_edges_to_routing_table(vec![new_edge.clone()]);
 
         let network_metrics = self.network_metrics.clone();
+        let run_later_span = tracing::trace_span!(target: "network", "RequestRoutingTableResponse");
         near_performance_metrics::actix::run_later(
             ctx,
             WAIT_FOR_SYNC_DELAY.try_into().unwrap(),
```

Comment:
cc @pompon0 this code looks somewhat weird to me. Essentially, it does

```
sleep(1s)
let known_edges = act.network_graph.read().edges().values().cloned().collect();
sleep(1s)
everything else
```

It looks like threre's a needless `run_later` here. 

Nothing actionable, just a weirdness I've noticed.

Reply:
looks weird indeed. It was apparently introduced here: https://github.com/near/nearcore/commit/1a2d8a4c339ea9a302bdce6de9e6f68e2b098619#diff-3ee228763dc067bf0f25ae418169bb3cf88ca7cf5e2d1883d13a3bd33fd5a8e3L727